docs/stencil-docs/installing-stencil-cli/installing-stencil.mdx

```diff
@@ -111,7 +116,7 @@ If you're a pro at installing and configuring Python and Node.js environments on
 **Required Dependencies:**
 * [Git](https://git-scm.com/downloads) - required to run npm install
 * [Python 2.7.x](https://www.python.org/downloads/) - required to build some dependencies
-* [Node.js 14 and npm](https://nodejs.org/en/download/releases/) - later versions not currently supported on Windows
+* [Node.js 18.15.0 and npm](https://nodejs.org/en/download/releases/) - later versions not currently supported on Windows

```

Comment:
```suggestion
* [Node.js 18.15.0 and npm](https://nodejs.org/en/download/releases/)
```
@jairo-bc I suppose this should be updated then ^

Reply:
right, not sure from where did it come to